c2rust-analyze/src/util.rs

```diff
@@ -276,7 +278,7 @@ fn builtin_callee(tcx: TyCtxt, did: DefId) -> Option<Callee> {
             if tcx.impl_trait_ref(parent_did).is_some() {
                 return None;
             }
-            let parent_impl_ty = tcx.type_of(parent_did);
+            let parent_impl_ty = EarlyBinder(tcx.type_of(parent_did)).subst(tcx, substs);
```

Comment:
could you comment on why this is necessary

Reply:
This was from #918, which I mistakenly merged into this branch instead of into `master`

Reply:
ok, I'd still like to know why it was done. I should have commented on #918

Reply:
In #918, I assumed it was to get the actual type instead of the generic type.